### PR TITLE
refact(storage): removing unused return in saveEventSql() function

### PR DESCRIPTION
**Pull request by isaqueveras**

### CODE REVIEW

File: storage/postgresql/save_test.go [hunk 1]
```diff
@@ -157,21 +152,16 @@ func TestSaveEventSql(t *testing.T) {
 				Content:   "test",
 				Sig:       "sig",
 			},
-			query: `INSERT INTO event (
-	id, pubkey, created_at, kind, tags, content, sig)
-	VALUES ($1, $2, $3, $4, $5, $6, $7)
-	ON CONFLICT (id) DO NOTHING`,
+			query:  `INSERT INTO event (id, pubkey, created_at, kind, tags, content, sig) VALUES ($1, $2, $3, $4, $5, $6, $7) ON CONFLICT (id) DO NOTHING`,
```
Comment (fiatjaf) [inline storage/postgresql/save_test.go:155]: why these changes? better leave it like it was, no?

Comment (isaqueveras) [reply storage/postgresql/save_test.go:155]: It could be. The force of habit of leaving everything in a single line.

File: storage/postgresql/save.go [hunk 2]
```diff
@@ -74,16 +74,9 @@ func deleteBeforeSaveSql(evt *nostr.Event) (string, []any, bool) {
...
-	ON CONFLICT (id) DO NOTHING`
-
-	var (
-		tagsj, _ = json.Marshal(evt.Tags)
-		params   = []any{evt.ID, evt.PubKey, evt.CreatedAt, evt.Kind, tagsj, evt.Content, evt.Sig}
-	)
-
-	return query, params, nil
+func saveEventSql(evt *nostr.Event) (string, []any) {
+	const query = `INSERT INTO event (id, pubkey, created_at, kind, tags, content, sig) VALUES ($1, $2, $3, $4, $5, $6, $7) ON CONFLICT (id) DO NOTHING`
```
Comment (bndw) [inline storage/postgresql/save.go:78]: I don't see the value in removing the error return, or changing these newlines and groupings. The error is there because we're currently swallowing and might want to not do that.
